save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsOrganizationsMenuView.kt

```diff
@@ -2,21 +2,24 @@ package com.saveourtool.save.frontend.components.views.usersettings
 
 import com.saveourtool.save.domain.Role
 import com.saveourtool.save.frontend.components.basic.cardComponent
+import com.saveourtool.save.frontend.externals.fontawesome.faTrashAlt
+import com.saveourtool.save.frontend.externals.fontawesome.fontAwesomeIcon
 import com.saveourtool.save.v1
-import csstype.ClassName
 
-import react.FC
+import csstype.ClassName
+import react.*
 import react.dom.html.ReactHTML.a
 import react.dom.html.ReactHTML.div
 import react.dom.html.ReactHTML.h1
 import react.dom.html.ReactHTML.img
 import react.dom.html.ReactHTML.li
 import react.dom.html.ReactHTML.ul
 
-@Suppress("MISSING_KDOC_TOP_LEVEL", "TOO_LONG_FUNCTION")
+@Suppress("MISSING_KDOC_TOP_LEVEL", "TOO_LONG_FUNCTION", "LongMethod")
 class UserSettingsOrganizationsMenuView : UserSettingsView() {
     private val organizationListCard = cardComponent(isBordered = false, hasBg = true)
-    override fun renderMenu(): FC<UserSettingsProps> = FC { props ->
+
+    override fun renderMenu(): FC<UserSettingsProps> = VFC {
```

Comment:
Fix the type pls

Reply:
props is not used anywhere, so I replaced it with VFC